### PR TITLE
add sc detection to vm snapshots

### PR DESCRIPTION
**Pull request by glekner**
Screen Shot 2020-12-03 at 13 30 42

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/modals/snapshot-modal/snapshot-modal.tsx [hunk 1]
```diff
@@ -62,20 +82,78 @@ const SnapshotsModal = withHandlePromise((props: SnapshotsModalProps) => {
...
+                </StackItem>
+              )}
+              {hasUnsupportedVolumes && (
+                <StackItem>
+                  <Alert
+                    variant={AlertVariant.warning}
+                    isInline
+                    title={t(
+                      'kubevirt-plugin~The following disk(s) will not be included in the snapshot for the following reason(s)',
+                    )}
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/modals/snapshot-modal/snapshot-modal.tsx:116]:
```suggestion
                    title={t(
                      'kubevirt-plugin~The following disk will not be included in the snapshot for the following reason',
                    {count: unsupportedVolumes?.length})}
```

**Comment by yaacov** [comment]
/cancel lgtm

PR has merge conflicts

**Comment by rawagner** [comment]
@yaacov rebased https://github.com/openshift/console/pull/7418

/close

**Pull request closed without merging**